scalafix/README.md

```diff
@@ -50,3 +50,13 @@ sbt scalafix github:typelevel/cats/v1.0.0
 - [ ] foldLeftM is removed from Free, use foldM on Foldable instead, see #1117 for detail.
 
 - [ ] iteratorFoldM was removed from Foldable due to #1716
+
+
+## To test scala fix
+
+```bash
+sbt ;coreJVM/publishLocal;coreFree/publishLocal
```

Comment:
Is this redundant with the other scalafix PR?

Reply:
Are you referring the PR that add this to travis? This is to add it to readme so people know how to run the test locally.